main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+// HomeHandler handle GET request to the root endpoint
+func HomeHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
```

Comment:
@tomipaul I think we need to have more discussions on this handler function

Reply:
@mikey2020 I'm proceeding to merge this consequent to our discussion.